NRzure/install/install.yml

```diff
@@ -0,0 +1,14 @@
+id: template
```

Comment:
This will need to be filled out to allow this quickstart to be "installed" in the platform. I would recommend using a `link` directive and sending them over to a documentation page.